pypeit/spectrographs/keck_deimos.py

```diff
@@ -210,10 +210,12 @@ def get_detector_par(self, det, hdu=None):
         ))
 
         if hdu is not None:
+            amp = self.get_meta_value(self.get_headarr(hdu), 'amp')
+            amp_folder = "ampA" if amp == 'SINGLE:A' else "ampB"
```

Comment:
Just to future proof, we should raise an exception if `amp=='DUAL:A+B'` here.

Reply:
I added an error message here if `amp=='DUAL:A+B'`